templeton/src/main/resources/templates/questions/Evaluation.html

```diff
@@ -7,17 +7,481 @@
 <head th:replace="fragment/common :: headerFragment"/>
 <body>
 <section th:replace="fragment/common :: topFragment"/>
-<section th:replace="fragment/common :: titleBar (title='Example Web Form')"/>
+<section th:replace="fragment/common :: titleBar (title='Evaluating the Program')"/>
 
 <section id="content" class="clearfix">
     <div class="container">
         <div class="row">
             <div class="span12">
 
 
-                <form th:action="@{/questions/MyWebForm}" method="POST">
-                    <label for="flowers">Do you like flowers? </label>
-                    <input type="checkbox" id="flowers" name="values"/>
+                <form th:action="@{/questions/Evaluation}" method="POST">
+
+                    <p> Thank you for taking the time to go through our program!
+                        We are interested in your feedback and reactions to the training programs, assessments, and other aspects of the website.
+                        Please answer the following questions openly and honestly. Your feedback will help us improve the program.
+                        *Note that assessment sessions were sessions in which you completed questionnaires about your feelings and thoughts.
+                        Training sessions were sessions in which you either completed words at the end of sentences OR saw two words on the screen
+                        and then a letter that you had to identify by pressing a button on the keyboard.
+                    <p/>
+
+                    <br/>
+                    <br/>
+
+                    <p> How helpful did you find the web program for reducing your anxiety? </p>
+                    <label> Not at all
+                        <input type="radio"  name="Helpful" value = "0"/>
+                    </label>
+                    <label> Slightly
+                        <input type="radio"  name="Helpful" value = "1"/>
+                    </label>
+                    <label> Somewhat
+                        <input type="radio"  name="Helpful" value = "2"/>
+                    </label>
+                    <label> Mostly
+                        <input type="radio"  name="Helpful" value = "3"/>
+                    </label>
+                    <label> Very
+                        <input type="radio"  name="Helpful" value = "4"/>
+                    </label>
+
+                    <label> Prefer Not to Answer
+                        <input type="checkbox"  name="NoAns_Helpful" />
+                    </label>
+
+                    <br/>
```

Comment:
We should not use <br/> tags if at all possible.  Better to surround the questions with a <div class="question"> tag which will help define the structure of the document and allow us more room to make adjustments in the CSS that are consistent everywhere.